gammapy/irf/edisp/map.py

```diff
@@ -538,3 +540,50 @@ def peek(self, figsize=(15, 5)):
 
         """
         self.get_edisp_kernel().peek(figsize)
+
+
+def _drm_to_edisp(filename, geom, geom_true, geom_psf):
```

Comment:
This function would be worth adding in an independent PR. We can support it as a new data format for DL4 IRFs similar to `gtpsf` for the PSF, see here: https://github.com/gammapy/gammapy/blob/main/gammapy/irf/core.py#L792